dashboard-lib/core-node-components/platform.go

```diff
@@ -0,0 +1,29 @@
+package core_node_components
+
+type PlatformOpts struct {
+	// Platform is infrastructure deployment platform: docker or k8s
+	Platform     string
+	LabelFilters map[string]string
+	LabelFilter  string
+	LegendString string
+	LabelQuery   string
+}
+
+// PlatformPanelOpts generate different queries for "docker" and "k8s" deployment platforms
+func PlatformPanelOpts() PlatformOpts {
+	po := PlatformOpts{
+		LabelFilters: map[string]string{
+			"env":          `=~"${env}"`,
+			"cluster":      `=~"${cluster}"`,
+			"blockchain":   `=~"${blockchain}"`,
+			"product":      `=~"${product}"`,
+			"network_type": `=~"${network_type}"`,
```

Comment:
I can't access grafana through the vpn currently, so sorry for the trivial question - what values does network_type take on?

Reply:
mainnet or testnet